lib/IDE/ExprContextAnalysis.cpp

```diff
@@ -1,4 +1,4 @@
-//===--- ExprContextAnalysis.cpp - Expession context analysis -------------===//
+//===--- ExprContextAnalysis.cpp - Expression context analysis -------------===//
```

Comment:
Could you adjust this to continue being 80 columns?
```suggestion
//===--- ExprContextAnalysis.cpp - Expression context analysis ------------===//
```